templates/pages/checkout.html

```diff
@@ -1 +1,35 @@
-brand.html
+<!doctype html>
+<html>
+<head>
+    <base href="/">
```

Comment:
We should probably inject this tag as part of `{{{ checkout_content }}}` instead of asking the theme developer to include it. AFAIK `<base href="/">` is only required because of `ng-checkout` and we should therefore hide this implementation detail.